pyzeebe/errors/pyzeebe_errors.py

```diff
@@ -24,3 +24,16 @@ def __init__(self, task_type: str):
 
 class MaxConsecutiveTaskThreadError(PyZeebeError):
     pass
+
+
+class BusinessError(PyZeebeError):
+    """
+    Exception that can be raised with a user defined code,
+    to be caught later by an error event in the Zeebe process
+    """
+    def __init__(self, error_code: str) -> None:
+        super().__init__(f"Business error with code {error_code}")
+        self.error_code = error_code
+
+    def __repr__(self) -> str:
+        return str({"error_code": self.error_code})
```

Comment:
Please add a newline at the end of the file :)

Reply:
I sorry, forgot to activate my formatter for this project haha